x-pack/plugins/monitoring/server/routes/api/v1/alerts/enable.ts

```diff
@@ -26,7 +26,6 @@ export function enableAlertsRoute(server: MonitoringCore, npRoute: RouteDependen
     async (context, request, response) => {
       try {
         const alertingContext = await context.alerting;
-        const infraContext = await context.infra;
```

Comment:
ℹ️ This is the only place the infra plugin was used on the server side - it filled in the space it in some server info logs about rule creation.

Theoretically it would be possible to add the spaces plugin as dependency to get the same information, but it doesn't seem worth it doing so as this is strictly about info logging. So I just removed the space id from the log line